Fabric.Authorization.AccessControl/src/app/models/fabricPrincipal.model.ts

```diff
@@ -6,5 +6,7 @@ export class FabricPrincipal {
     public lastName: string;
     public principalType: string;
 
```

Comment:
Very minor - if `middleName` is blank, there will be 2 spaces between `firstName` and `lastName` in the UI.

Reply:
good catch i actually meant to remove this